src/azul/service/manifest_service.py

```diff
@@ -604,9 +608,19 @@ def entity_type(self) -> str:
     def source_filter(self) -> SourceFilters:
         return [
             *super().source_filter,
-            'contents.files.related_files'
+            'contents.files.related_files',
+            'contents.files.drs_path'
         ]
 
+    @cached_property
+    def manifest_config(self) -> ManifestConfig:
+        config = deepcopy(super().manifest_config)
+        assert isinstance(config, dict)
```

Comment:
If this is to work around a type warning, consider typing.cast instead since it is more explicit.